docs/rules/replace-j-unit-timeout-annotation-property.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Replace JUnit Timeout Annotation Property with assertTimeout
+ruleId: ReplaceJUnitTimeoutAnnotationProperty
+since: 3.26.0
+minJavaVersion: 8
+remediationCost: 5
+    
+description:
+    This rule aims to replace the @Test(timeout=...) annotation property with 'assertTimeout' in JUnit Jupiter.
+tags: ["Java 8", "Testing", "Lambda", "Readability"]
+---
+
+# Replace JUnit Timeout Annotation Property with assertTimeout
+
+[[toc]]
+
+## Properties
+
+<RuleProperties />
+
+## Description
+
+JUnit [Jupiter API](https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html) provides timeout assertions, i.e., assertions that execution of some code completes before a timeout exceeds. 
```

Comment:
I would change the link [Jupiter API](https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertTimeout-java.time.Duration-org.junit.jupiter.api.function.Executable-) which shows the first assertTimeout - method